src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProjectTypeRegistration.cs

```diff
@@ -1,34 +1,17 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.Runtime.InteropServices;
-
 using Microsoft.VisualStudio;
 using Microsoft.VisualStudio.Packaging;
 using Microsoft.VisualStudio.ProjectSystem;
 using Microsoft.VisualStudio.ProjectSystem.VS;
-using Microsoft.VisualStudio.Shell;
 
-// Register ourselves as a CPS project type
+// Visual Basic
 [assembly: ProjectTypeRegistration(
     projectTypeGuid: ProjectType.VisualBasic,
     displayName: "#1",                      // "Visual Basic"
```

Comment:
Can we make these nicer names, given that we'll have a bunch of them in the same assembly now? Or do they have to be positional?

Reply:
We cannot, they must be integers.

Reply:
The design is based on Win32 resources and managed resources were kinda hacked on top of it.